### PR TITLE
Add gps files and fix typo

### PR DESCRIPTION
**Pull request by spxiwh**
Here's a pull request to fix a typo I spotted in data_C02.

I also added GPS time files for all chunks.

### CODE REVIEW

File: O2clean/pipeline/gps_times_O2_analysis_10.ini [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+; Sat Mar 18 20:00:00 GMT 2017
+start-time = 1173902418
+
+; Mon Mar 27 11:23:02 GMT 2017
+end-time = 1174649000
+
+[results_page]
+analysis-title = "PyCBC offline search"
+analysis-subtitle = "O2 Analysis 10 v1.6.8 (Mar 18 - Mar 27)"
```
Comment (tdent) [inline O2clean/pipeline/gps_times_O2_analysis_10.ini]: we should probably remove the version tag statements too ..

File: O2clean/pipeline/gps_times_O2_analysis_18.ini [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+; Sat Jul 15 00:00:00 GMT 2017
+start-time = 1184112018
+
+; Thu Jul 27 19:00:00 GMT 2017
+end-time = 1185217218
+
+[results_page]
+analysis-title = "PyCBC offline search"
+analysis-subtitle = "O2 Analysis 18 full run (C00, final VDF, Jul 15 - 27)"
```
Comment (tdent) [inline O2clean/pipeline/gps_times_O2_analysis_18.ini]: remove 'C00, final VDF'

**Comment by spxiwh** [comment]
Stripping config files as Tom suggests

**Comment by dethodav** [comment]
Looks good, merged.

**Pull request merged**